torch/csrc/distributed/autograd/functions/recvrpc_backward.cpp

```diff
@@ -0,0 +1,33 @@
+#include <torch/csrc/distributed/autograd/functions/recvrpc_backward.h>
+#include <ATen/core/functional.h>
+
+namespace torch {
+namespace distributed {
+namespace autograd {
+
+using torch::autograd::Variable;
+
+torch::autograd::variable_list RecvRpcBackward::apply(
+    torch::autograd::variable_list&& grads) {
+  // Get size and options for tensors.
+  at::IntArrayRef sizes;
+  at::TensorOptions o;
+  for (auto v : grads) {
+    if (v.defined()) {
+      sizes = v.sizes();
+      o = static_cast<at::Tensor>(v).options();
+      break;
+    }
+  }
+
+  // Put in zeros for grads that are not defined.
+  auto output_grads = c10::fmap(grads, [&](const Variable& v) {
+    return (v.defined() ? v : Variable(at::zeros({}, o).expand(sizes)));
```

Comment:
Future perf optimization: Is it possible to add these zero grad on the `SendRpcBackward` function, so that we don't have to communicate them over the wire.?

Reply:
https://github.com/pytorch/pytorch/issues/26446